app/models/katello/input_template_renderer.rb

```diff
@@ -0,0 +1,7 @@
+class InputTemplateRenderer
+  include UnattendedHelper
+
+  def errata(id)
+    Katello::Erratum.find_by_errata_id(id).attributes.slice!('errata_id', 'created_at', 'updated_at')
```

Comment:
Just thought no one is going to be interested having 'errata_id', 'created_at' & 'updated_at', But I guess if we take db ids & errata ids I would make sense to return the errata_id. I dont see the point of returning 'created_at' & 'updated_at' ?